### ISSUE TITLE
Cannot update a vulnerability object with new CVSS3 score information

### ISSUE THREAD

**Issue by sweet-mentat** [LuatixHQ/opencti#3431]
## Description

Sending a bundle containing a Vulnerability object with OpenCTIConnectorHelper.send_stix2_bundle function, causes this error to show up in the connector page (red error button): {'name': "Cannot read properties of null (reading 'multiple')", 'message': "Cannot read properties of null (reading 'multiple')"}
On the platform, I found this error:
{"category":"APP","error":{"stacktrace":["TypeError: Cannot read properties of null (reading 'multiple')","at loadElementMetaDependencies (/opt/opencti/build/src/database/middleware.js:425:33)","at processTicksAndRejections (node:internal/process/task_queues:95:5)"]},"inner_relation_creation":0,"level":"warn","message":"API Call","operation":"VulnerabilityAdd","operation_query":"mutation VulnerabilityAdd($input:VulnerabilityAddInput){vulnerabilityAdd(input:$input){id standard_id entity_type parent_types}}","size":573,"time":63,"timestamp":"2023-05-04T15:09:51.388Z","type":"WRITE_ERROR","user":{"applicant_id":"xxxxxxxxxxx","call_retry_number":"1","ip":"xxx.xxx.xxx.xxx","user_id":"xxxxxxxxxxxxxxx","version":"5.7.3"}

I also tried using the Global Import page, pasting the bundle into the textbox and importing it with the ImportFileStix2 importer. I received the same error.

I also received no error when creating a new Vulnerability object that did not previously exist in OpenCTI.
I received no error when submitting a Vulnerability object in which the information did not differ from what was already in OpenCTI.
I receive an error if submitting a Vulnerability with new information, and if the update parameter for the send_stix2_bundle function is True or False.
I have also tried updating a vulnerability using pycti.Vulnerability.create(). I get no error, but no updates are made either.
Lastly, I get an error when attempting to update a Vulnerability object in the GUI by changing one of the CVSS3 options.

## Environment

OS (where OpenCTI server runs): hub.docker.com/opencti/platform:5.7.3
OpenCTI version: 5.7.3
Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:

1. In the GUI, go to an existing Vulnerability object.
2. Click the edit icon.
3. Click on Details.
4. Change at least one of the CVSS3 entries.
5. Click out of it to save.
## Expected Output

An updated Vulnerability object
## Actual Output

I get a pop up with the error:  "Cannot read properties of null (reading 'multiple')"

## Screenshots (optional)
![opencti_vulnerability_edit_error](https://github.com/OpenCTI-Platform/opencti/assets/77708915/19b367c0-cdf3-4e7b-b7f7-b6f516050266)

**Comment by SarahBocognano** [comment]
Hello @sweet-mentat, I've tested the issue and it seems that this bug has been corrected since the 5.7.3 version. I suggest that maybe you migrate to the new version of OpenCTI and tell us if the bug is still present on the platform :)

**Comment by Kedae** [comment]
No activity since June 1, please reopen a new issue with more information if the problem occurs again

**Comment by sweet-mentat** [comment]
I did end up finding the problem, which was on my end, not yours. While developing a connector that creates and modifies both note and vulnerability objects, we upgraded to 5.7.3 and turned on the sharing capability. I had created a user for my connector and accidentally assigned it to an organization with no members. Thus, I could create objects, but not modify vulnerability objects with any user, including a user with admin privileges. I could, however, modify note objects, and wanted to make sure that this was known.
